core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -220,17 +220,14 @@ impl<C: LikeClusterInfo, R: ReceiveAndBuffer> SchedulerController<C, R> {
             &mut error_counters,
         );
 
-        let fee_check_results: Vec<_> = check_results
+        for ((check_result, tx), result) in check_results
             .into_iter()
             .zip(transactions)
-            .map(|(result, tx)| {
-                result?; // if there's already error do nothing
-                Consumer::check_fee_payer_unlocked(bank, *tx, &mut error_counters)
-            })
-            .collect();
```

Comment:
yeah idk why I would have collected here, so dumb

Reply:
new you would never